Competition/src/main/cpp/RobotContainer.cpp

```diff
@@ -17,6 +17,7 @@ RobotContainer::RobotContainer() {
 void RobotContainer::ConfigureButtonBindings() {
     m_drivetrain.setController(&m_GamepadDriver);
     m_arm.setController(&m_GamepadOperator);
+    m_hopper.setController(&m_GamepadOperator, &m_GamepadOperator);
```

Comment:
Should this be driver and operator, instead of 2 operators?

Reply:
Ah, yes it should ill push that change